docs/style.md

```diff
@@ -180,9 +199,15 @@ Developers may still use external gem dependencies but should vendor it
 with their plugins or resource packs.
 
 
-### 8. Avoid debugging calls (in production)
+### 9. Avoid debugging calls (in production)
 
 Reason: One of the best way to develop and explore tests is the interactive debugging shell `pry` (see the section on "Interactive Debugging with Pry" at the end of this page). However, after you finish your profile make sure you have no interactive statements included anymore. Sometimes interactive calls are hidden behind conditionals (`if` statements) that are harder to reach. These calls can easily cause trouble when an automated profiles runs into an interactive `pry` call that stops the execution and waits for user input.
 
 Avoid: `binding.pry` in production profiles
 Use: Use debugging calls during development only
+
+Also you may find it helpful to use the inspec logging interface:
+
+```ruby
+Inspec::Log.info("Hi")
```

Comment:
/me squints eyes at quote usage

I see where you stand :stuck_out_tongue_closed_eyes: